docs/fundamentals/dashboard/configuration.md

```diff
@@ -0,0 +1,114 @@
+---
+title: .NET Aspire dashboard configuration
+description: .NET Aspire dashboard configuration options
+ms.date: 03/13/2024
+ms.topic: reference
+---
+
+# Dashboard configuration
+
+The dashboard is configured when it starts up. Configuration includes frontend and OTLP addresses, the resource service endpoint, authentication, telemetry limits and more.
+
+There are a number of ways to provide configuration:
+
+- Command line arguments.
+- Environment variables. The `:` delimiter should be replaced with double underscore (`__`) in environment variable names.
+- Optional JSON configuration file. The `DOTNET_DASHBOARD_CONFIG_FILE_PATH` setting can be used to specify a JSON configuration file.
+
+Example of configuring the dashboard when started from a Docker container:
+
+```bash
+docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard \
+    -e DASHBOARD__TELEMETRYLIMITS__MAXLOGCOUNT='1000' \
+    -e DASHBOARD__TELEMETRYLIMITS__MAXTRACECOUNT='1000' \
+    -e DASHBOARD__TELEMETRYLIMITS__MAXMETRICSCOUNT='1000' \
+    mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.6
+```
+
+Alternatively, configuration in JSON configuration file that is specified using `DOTNET_DASHBOARD_CONFIG_FILE_PATH`:
```

Comment:
Are we going show the end to end here? Using a bind mount to load this configuration into the container? Giving people something to copy and paste like above is really nice.

Reply:
Maybe. I was just writing some docs saying it requires a bind mount.

There is already a lot in this PR. I'll leave it as a follow work.